WcaOnRails/app/javascript/edit-schedule/SchedulesEditor/calendar-utils.js

```diff
@@ -96,7 +110,7 @@ function ambiguousMomentToIsoString(editor, momentObject) {
   let tz = venue.timezone;
   // Take the moment and "concatenate" the UTC offset of the timezone at that time
   // momentObject is a FC (ambiguously zoned) moment, therefore format() returns a zone free string
-  return moment.tz(momentObject.format(), tz).format();
+  return moment.tz(momentObject.format(), tz).utc().format();
```

Comment:
Why is that necessary?

Reply:
It's not necessary, but I did it for the sake of consistency. Previously it would return an activity with `startDate` and `endDate` as ISO strings with timezone, whereas initially it's always UTC.

Reply:
I wish I'd remember why I did it like that, but it seems good to me now.